extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers/controller.configurableService.dialog.js

```diff
@@ -29,6 +29,25 @@ configurableServiceDialogController.controller('ConfigurableServiceDialogControl
         }).then(applyDefaults(configDescriptionURI));
     }
 
+    var addESHServiceContextParameter = function(configDescription) {
+        var found = false;
+        angular.forEach(configDescription.parameters, function(parameter) {
+            if (parameter.name === 'esh.servicecontext') {
+                found = true;
+            }
+        })
+        if (!found) {
+            var eshServiceContextParameter = {
```

Comment:
Could you please add some description that guides the non-savvy user a little on what to do with this "Service Context" field?